src/middlewares/user.midleware.js

```diff
@@ -0,0 +1,13 @@
+import ResponseService from '../services/response.service';
+import UserService from '../services/user.service';
+
+const checkIfEmailExist = async (req, res, next) => {
+	const user = await UserService.findUserByAttribute({ email: req.body.email });
+	if (user) {
+		ResponseService.setError(409, 'email is already exist');
+		return ResponseService.send(res);
+	}
+	next();
+};
+
+export default checkIfEmailExist;
```

Comment:
can you export this function as a named function? Tips use `export` instead of `export default` because in this file they will be other functions

Reply:
solved